ipn/local.go

```diff
@@ -1338,13 +1339,15 @@ func (b *LocalBackend) enterState(newState State) {
 
 	switch newState {
 	case NeedsLogin:
+		systemd.Status("login failed as %s", b.activeLogin)
 		b.blockEngineUpdates(true)
 		fallthrough
 	case Stopped:
 		err := b.e.Reconfig(&wgcfg.Config{}, &router.Config{})
 		if err != nil {
 			b.logf("Reconfig(down): %v", err)
 		}
+		systemd.Status("stopped, please run 'tailscale up' to log in")
```

Comment:
drop the `please`. space is limited. I'd also separate use a semicolon to separate the stopped with the restorative action.

    `stopped; run 'tailscale up' to log in`